dockerfiles/Dockerfile

```diff
@@ -64,3 +64,14 @@ RUN microdnf upgrade -y && \
     cp /tmp/work/run-integration.py /run-integration.py
 
 CMD [ "/run-integration.py" ]
+
+FROM prod-image as fedramp-prod-image
+ENV FEDRAMP_OC_VERSION=4.12.46
+
+# FedRamp is typically on older versions of OpenShift clusters than commercial so we build a new version of
+# the prod image just replacing oc and kubectl with our own version
+RUN curl -sfL https://mirror.openshift.com/pub/openshift-v4/clients/ocp/${FEDRAMP_OC_VERSION}/openshift-client-linux.tar.gz \
+    -o oc.tar.gz && \
+    tar -zvxf oc.tar.gz && \
+    mv oc kubectl /usr/local/bin && \
+    rm oc.tar.gz README.md
```

Comment:
Due to overlay2 fs, 2 versions of `oc` will exist in different layers, `4.12.46` will be the file used eventually, but total image size will include binaries for both versions.

Reply:
Darn yeah that's right I forgot about that.